Sources/SwifterSwift/Foundation/URLSessionExtensions.swift

```diff
@@ -0,0 +1,33 @@
+// URLSessionExtensions.swift - Copyright 2023 SwifterSwift
+
+#if canImport(Foundation)
+import Foundation
+
+#if canImport(FoundationNetworking)
+import FoundationNetworking
+#endif
```

Comment:
If these imports fail then the rest of the code won't compile. Look at how it's done in other files, were the `#endif` is added after the code that requires these libraries.
```suggestion
#if canImport(Foundation) && canImport(FoundationNetworking)
import Foundation
import FoundationNetworking
```

Reply:
Hello @guykogus 
There is no `#if canImport(Foundation) && canImport(FoundationNetworking)` in SwifterSwift because it makes an extension available for Linux only. In other files I see
```
#if canImport(Foundation)
import Foundation

#if canImport(FoundationNetworking)
import FoundationNetworking
#endif

extension Something {
}

#endif
```

Reply:
Oh you're right, I actually wasn't fully familiar with `FoundationNetworking`. Thanks :)